debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/JavaVersionHelper.java

```diff
@@ -53,4 +53,22 @@ public static boolean ensureJava9AndUp(J9JavaVMPointer vm, PrintStream out) thro
 		}
 		return true;
 	}
+	
+	/**
+	 * Returns true if the Java version is Java11 and up.
+	 * @param vm J9JavaVMPointer
+	 * @param out The output print stream
+	 * @return if the Java version is Java11 and up or not.
+	 * @throws CorruptDataException
+	 */
+	public static boolean ensureJava11AndUp(J9JavaVMPointer vm, PrintStream out) throws CorruptDataException 
+	{
+		int javaVersion = vm.j2seVersion().bitAnd(J2SE_SERVICE_RELEASE_MASK).intValue() >> J2SE_JAVA_SPEC_VERSION_SHIFT;
+		if (javaVersion < J2SE_V11) {
+			out.printf("This command only works with Java version 11 or higher%n"
+					+ "The current VM Java version is: %s%n", javaVersion);
```

Comment:
Nit: These sentences should each end with a period (you are talking to a human).